### DISCUSSION TITLE
A few questions about v2

### DISCUSSION THREAD

**Discussion by MisterSheeple** [younesaassila/ttv-lol-pro#180] [General]
First, what exactly does passport mode do? It says "This option enables advanced proxying to prevent ads" but what exactly does that entail?

Also, what data is sent when you have a proxy under "Proxy all requests"? Does this include personally identifiable info such as the user's Twitch token or IP address?

**Comment by younesaassila** [comment]
Passport proxies `passport.twitch.tv` and `usher.ttvnw.net` requests.

"Proxy ad requests" and "Proxy all requests" proxy your Twitch playback token.
Hidden option "Laissez-passez" proxies all data included in Twitch GraphQL requests.

However, since the extension is using standard HTTP proxies, the connection between you and Twitch is encrypted (HTTPS) and the proxies cannot read any data being passed through it except the domain name (like gql.twitch.tv or passport.twitch.tv)

**Reply by MisterSheeple** [reply]
Thanks, closing.
